lib/hubstats/github_api.rb

```diff
@@ -63,7 +63,7 @@ def self.get_repos
       repos
```

Comment:
Umm, I don't think there is any logic for adding organizations through webhooks. At least @elliothursh didn't put any in....